packages/components/notification/Notification.mdx

```diff
@@ -30,7 +33,7 @@ Notifications give immediate feedback about an action triggered or experienced b
 
 ```jsx
```

Comment:
What's the thing about adding `static=true` again?

Do we need it or not?

Reply:
That is for not rendering the example and only showing the code snippet, I only used that for the codes about the API as they don't get rendered.
So it depends if we want to have the example working on the docs or only the code

Reply:
Ahhhhhhhhhhhhh I got it
I will have to change my stuff hahahaha

Thanks 🙏